cmd/add/add.go

```diff
@@ -36,7 +39,10 @@ webman add go@18.0.0
 webman add go zig rg
 webman add go@18.0.0 zig@9.1.0 rg@13.0.0`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		utils.Init()
```

Comment:
Looks like the `utils.Init()` function is removed here, but on line 52 it references utils.RecipeDirFlag, which would always be `""` if not using `Init()`. How does that functionality work now?

Reply:
This was mostly a clarifying question, but I do see that those changes happen in config.Load().

Reply:
It was redundant because it gets called over at https://github.com/candrewlee14/webman/blob/65477862d527e14a1fa8331cd282940033b2e7a6/cmd/root.go#L90